shoop/core/models/products.py

```diff
@@ -535,6 +535,10 @@ def get_all_package_children(self):
             ProductPackageLink.objects.filter(parent=self).values_list("child", flat=True)
         ))
 
+    def get_public_media(self):
+        from .product_media import ProductMedia
+        return ProductMedia.objects.filter(product=self, enabled=True, public=True)
```

Comment:
Should use the `self.media` related manager:
```python
return self.media.filter(enabled=True, public=True)
```
Though a custom manager might be good addition too, i.e. this would be just:
```python
return self.media.enabled().public()
```
That would allow redefining public/enabled in the future (e.g. to make public() check if the current time is in date range or whatever).